### ISSUE TITLE
{Freeway-Bug}[Adaptive Cards- Product Video] :  There should be a functionality to customize font color, size, type etc. for ‘Captions’ in video player. 

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#9025]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/
OS Version: Dev (OS Build 27695.1000)
Browser Version: Version 130.0.2849.1 (Official build) dev (64-bit)
Screen Reader: Narrator

**Repro steps:**
1. Open the above URL in the edge dev browser.
2. Navigate to the 'Sample' option and press enter key to activate it.
3. Press tab key and navigate to the 'Product Video' link and press enter key to activate it.
4. Press tab key and navigate to Play button and press enter key to activate play button.
5. Observe the issue.

**Actual Result:**
Users don’t have the option to change the color, font size, width, height, etc., of the captions provided in video player present on the Adaptive Card application. 

There should be a feature to change/update these things for caption:
Font family
Font opacity
Font casing (some prefer all upper-case)
Font size
Font color
Background color (including none) 
Background opacity
Number of lines 
Positioning
Outlining

**Expected Result:**
The above customizations are highly desirable, this should be applicable to ‘Captions’ provided in video player present on the application. Where user controls are provided for volume adjustment, the user controls for customizing captions should be provided at the same menu level.

**Example:**
 PFA for reference: [Can You Save Taxes By Transferring Money Into Your Wife's Account ? | Watch (msn.com)](https://www.msn.com/en-in/video/money/can-you-save-taxes-by-transferring-money-into-your-wife-s-account/vi-AA1hUwDy?ocid=entnewsntp&cvid=a662a34de2ac4188a9c7aabe1b92cca4&ei=12) 

**Note:**
Same issue is repro throughout the application for the video player.
 
**User impact:**
Deaf low vision users who want to characterize the captions (including, for example, giving users the ability to change text size, text font, and color) would face difficulty seeing the options if there is no option to characterize the captions.
Where information and communication technology (ICT) displays captions, it shall provide a way for the user to adapt the displayed characteristics of captions to their individual requirements, 

**MAS Reference link:**
[Caption characteristics - Liquid (microsoft.com)](https://liquid.microsoft.com/Web/Object/Read/ms.accessibility/Requirements/07.01.04)
 
Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

![There should be a functionality to customize font color, size, type etc  for ‘Captions’ in video player](https://github.com/user-attachments/assets/8c6023f3-1a29-4ed8-a4d3-36bccbec8f60)

<img width="1923" alt="Example-Closed caption settings (1)" src="https://github.com/user-attachments/assets/b3793371-cb4f-4e61-a011-71f1bb261e88">

### Card JSON

```JSON
NA
```

**Comment by anna-dingler** [comment]
Hi @vagpt,

My team uses HTML's `video` element for this. Do you have any guidance on customizations here? I do not believe we have control over the caption styling.

**Comment by vagpt** [comment]
Hi @anna-dingler,

Could you please provide us new GA Date?

**Comment by vagpt** [comment]
Closing this issue here and now it is tracking with below bug - 

https://github.com/microsoft/AdaptiveCards/issues/9141
